editor/src/messages/tool/tool_messages/artboard_tool.rs

```diff
@@ -196,6 +214,39 @@ impl ArtboardToolData {
 			dimensions: size.round().as_ivec2(),
 		});
 
+		let top_edge_resized = bounds.selected_edges.as_ref().map_or(false, |edges| edges.top);
+		let left_edge_resized = bounds.selected_edges.as_ref().map_or(false, |edges| edges.left);
+		let original_left_edge_x = bounds.bounds[0].x;
+		let original_top_edge_y = bounds.bounds[0].y;
+
+		let translation_x = if left_edge_resized {
+			let delta_x = min.x.round() - original_left_edge_x.round();
+			delta_x
+		} else {
+			0.0
+		};
+
+		let translation_y = if top_edge_resized {
+			let deltay_y = min.y.round() - original_top_edge_y.round();
+			deltay_y
+		} else {
+			0.0
+		};
+
+		let translation = DVec2::new(translation_x, translation_y);
+		let reverse_translation = DVec2::new(
+			if left_edge_resized { -translation.x } else { translation.x },
+			if top_edge_resized { -translation.y } else { translation.y },
+		);
+		let child_layers = self.get_child_layers_of_selected_artboard(document);
+		for layer in child_layers {
+			responses.add(GraphOperationMessage::TransformChange {
+				layer,
+				transform: DAffine2::from_translation((reverse_translation).into()),
+				transform_in: TransformIn::Local,
```

Comment:
This doesn't work properly when there is a transform node right before the artboard node.